inclusive-design/idrc
# feat: move JavaScript processing into Eleventy (resolves #49)

## Description

Adapts Eleventastic's JavaScript processing, removing the need for Laravel Mix.

## Steps to test

1. Check out branch.
2. Run `npm install` and `npm start`
3. Visit http://localhost:3000.

**Expected behavior:** Observe live reloading when changes are made to `/src/js/idrc.js`.

## Additional information

Not applicable.

## Related issues

Resolves #49.
